src/game/level.c

```diff
@@ -403,6 +429,20 @@ static bool Level_LoadItems(MYFILE *fp)
             }
 
             InitialiseItem(i);
+
+            // Calculate number of pickups in a level
+            for (int16_t j = 0; j < NUM_PICKUP_OBJ; j++) {
+                if (item->object_number == m_PickupObjs[j]) {
+                    m_LevelPickups++;
+                }
+            }
+
+            // Calculate number of killable objects in a level
+            for (int16_t j = 0; j < NUM_KILLABLE_OBJ; j++) {
+                if (item->object_number == m_KillableObjs[j]) {
+                    m_LevelKillables++;
+                }
+            }
```

Comment:
How this behaves with Pierre?

Reply:
Hmm in levels where you fight but don't kill him, it seems like the game doesn't let his HP go below half, so this will cause there to be extra unkillables for those levels. I'll need to check how the Pierre code is handled.